tests/unit/dom.ts

```diff
@@ -488,5 +488,36 @@ registerSuite({
 			assert.strictEqual(parent.children.length, 0);
 			assert.isNull(node.parentNode);
 		}
+	},
+
+	create: {
+		'correct element created'() {
+			let tagType = 'div';
+			let result = dom.create(tagType);
+			assert.equal(result.nodeName.toLowerCase(), tagType);
+		},
+
+		'attributes are set on element'() {
+			let args: CreateArgs = { foo: 'bar', baz: 'buz', attributes: { 'class': 'test' } },
+				element = dom.create('div', args);
+
+			assert.strictEqual((<any> element).foo, 'bar');
+			assert.strictEqual((<any> element).baz, 'buz');
+			assert.strictEqual(element.className, 'test');
+		},
+
+		'String children are appended to element'() {
+			let element = dom.create('div', null, ['<div id="test"></div>']);
+			assert.strictEqual(element.children.length, 1);
+			assert.strictEqual((<any> element.firstChild).id, 'test');
+		},
+
+		'Node children are appended to element'() {
+			let child = document.createElement('div');
+			child.id = 'test2';
```

Comment:
You don't really even need to add an id here, you should be able to just do a `strictEqual` to compare `element.firstChild` to `child`.

You should also test with more than one to actually test the "appended" part.